kolibri/core/device/api.py

```diff
@@ -312,6 +321,35 @@ def get_queryset(self):
             return UserSyncStatus.objects.none()
         return UserSyncStatus.objects.all()
 
+    def has_downloads(self, obj):
+        is_having_downloads = Subquery(
+            ContentDownloadRequest.objects.filter(
+                source_id=OuterRef("user_id"),
+                source_model=FacilityUser.morango_model_name,
+                reason__in=[
+                    ContentRequestReason.SyncInitiated,
+                    ContentRequestReason.AutoSync,
+                ],
```

Comment:
The use case for the information `has_downloads` is that the user initiated the download, which means the reason should be: `reason=ContentRequestReason.UserInitiated`

Additionally, `ContentRequestReason.AutoSync` doesn't exist.